src/commands/circleci-continue-long-job-cancel.yml

```diff
@@ -21,7 +25,7 @@ steps:
       working_directory: /tmp/apollo_internal_platform_orb/src/scripts/circleci-job-canceller
       command: |
         python3 -m pip install -r requirements.txt
-        python3 long_job_canceller.py $<< parameters.circleci-token-variable >> << parameters.org-repo-slug >> --commit --n-windows << parameters.n-windows >>
+        python3 long_job_canceller.py $<< parameters.circleci-token-variable >> << parameters.org-repo-slug >> --commit --n-windows << parameters.n-windows >> --ignore=<< parameters.ignore-jobs-containing >>
```

Comment:
same nit about the chance for all of these variable/argument names to be able to match. Less cognitive overheard overall